webui/react/src/pages/ExperimentList.tsx

```diff
@@ -727,6 +731,19 @@ const ExperimentList: React.FC<Props> = ({ project }) => {
         );
         ExperimentMoveModal.open();
       }
+      if (action === Action.RetainLogs) {
+        if (!settings?.row?.length) return;
```

Comment:
good to define concrete number like `== 0`

Reply:
done!